pkg/background/generate/generate.go

```diff
@@ -447,6 +451,12 @@ func applyRule(log logr.Logger, client *dclient.Client, rule kyverno.Rule, resou
 	common.ManageLabels(newResource, resource)
 	// Add Synchronize label
 	label := newResource.GetLabels()
+
+	// Add background gen-rule label if generate rule applied on existing resource
+	if rule.IsGenerateExisting() {
+		label["kyverno.io/background-gen-rule"] = rule.Name
+	}
```

Comment:
What is this label used for? Do we want to add to trigger resource as well?

Reply:
Its to filter out the resource which generated due to genExisiting was enabled , in case we want to delete them in single using a label